### ISSUE TITLE
[Bug]: layout recognizer failed for wrong boxes class type

### ISSUE THREAD

**Issue by ymsdu2004** [infiniflow/ragflow#4230]
### Is there an existing issue for the same bug?

- [X] I have checked the existing issues.

### RAGFlow workspace code commit ID

tag: v0.15.0

### RAGFlow image version

ragflow:v0.15.0

### Actual behavior

Traceback (most recent call last):
  File "/ragflow/deepdoc/vision/t_recognizer.py", line 190, in <module>
    main(args)
  File "/ragflow/deepdoc/vision/t_recognizer.py", line 48, in main
    layouts = detr(images, float(args.threshold))
  File "/ragflow/deepdoc/vision/recognizer.py", line 456, in __call__
    bb = self.postprocess(self.ort_sess.run(None, {k:v for k,v in ins.items() if k in self.input_names})[0], ins, thr)
  File "/ragflow/deepdoc/vision/recognizer.py", line 433, in postprocess
    return [{
  File "/ragflow/deepdoc/vision/recognizer.py", line 434, in <listcomp>
    "type": self.label_list[class_ids[i]].lower(),
IndexError: list index out of range

It got unexpected class_ids: [126  47 142  83   0  25]

<img width="1112" alt="1" src="https://github.com/user-attachments/assets/a52178a0-b57d-47c9-9f4a-c333023246ba" />

### Steps to reproduce

```Markdown
python deepdoc/vision/t_recognizer.py --inputs=<any_pdf_file> --threshold=0.2 --mode=layout --output_dir=./output
```

**Comment by ymsdu2004** [comment]
Has anyone encountered the same issue? Or could someone help provide a solution?

**Comment by KevinHuSh** [comment]
Update the model files in `rag/res/deepdoc` with [this](https://huggingface.co/InfiniFlow/deepdoc/tree/main).

**Comment by sc-lj** [comment]
@KevinHuSh i have upate model file,but this problem still 。

**Comment by KevinHuSh** [comment]
Have you restart the server?

**Comment by michaelice** [comment]
@KevinHuSh I am also encountering the same issue; I have updated the model code, but the problem remains unresolved.And I have restarted the service.

**Comment by michaelice** [comment]
Similarly，received unexpected class ids！
![image](https://github.com/user-attachments/assets/e2fb7a51-f411-4947-a5d7-b945a6b99736)

**Comment by ymsdu2004** [comment]
> Update the model files in `rag/res/deepdoc` with [this](https://huggingface.co/InfiniFlow/deepdoc/tree/main).
Thank you for your reply. I have updated the model, but the exception still occurs.

**Comment by fengjac** [comment]
I have the same problem. Unexpected class ids:

![image](https://github.com/user-attachments/assets/ab6b0deb-0347-4a92-9cde-b8fb9ca38304)

test file:
[1.pdf](https://github.com/user-attachments/files/18279132/1.pdf)
command:
```python
python deepdoc/vision/t_recognizer.py --inputs='./input/1.pdf' --threshold=0.2 --mode=layout --output_dir='./output'
```

**Comment by fengjac** [comment]
> > Update the model files in `rag/res/deepdoc` with [this](https://huggingface.co/InfiniFlow/deepdoc/tree/main).
> > Thank you for your reply. I have updated the model, but the exception still occurs.

Me too

**Comment by KevinHuSh** [comment]
After updating the model files in `rag/res/deepdoc`, you need to make sure all the `task_executor.py` be killed and restart.

**Comment by fengjac** [comment]
> After updating the model files in `rag/res/deepdoc`, you need to make sure all the `task_executor.py` be killed and restart.

Thanks @KevinHuSh. I have tried to reboot my computer, even cloned new project from github.
However, the exception still occurs.

**Comment by michaelice** [comment]
@KevinHuSh  #4330 After updating the code, a new issue arose. Replacing “recognizer” with “layout recognizer” resulted in inconsistent parameters, where “thr” was passed to “ocr_res”.
![image](https://github.com/user-attachments/assets/03738a47-80c2-4f3a-a6e1-6f0c635d9536)
![image](https://github.com/user-attachments/assets/b75c0527-3940-4dcd-a101-86350e1716a0)

**Comment by ymsdu2004** [comment]
after #4355 another error occurs @KevinHuSh 
`Traceback (most recent call last):
  File "/ragflow/deepdoc/vision/t_recognizer2.py", line 183, in <module>
    main(args)
  File "/ragflow/deepdoc/vision/t_recognizer2.py", line 41, in main
    layouts = detr(images, thr=float(args.threshold))
TypeError: LayoutRecognizer.__call__() missing 1 required positional argument: 'ocr_res'`

**Comment by fengjac** [comment]
> after #4355 another error occurs @KevinHuSh `Traceback (most recent call last): File "/ragflow/deepdoc/vision/t_recognizer2.py", line 183, in <module> main(args) File "/ragflow/deepdoc/vision/t_recognizer2.py", line 41, in main layouts = detr(images, thr=float(args.threshold)) TypeError: LayoutRecognizer.__call__() missing 1 required positional argument: 'ocr_res'` The `LayoutRecognizer.__call__` need a required parameter `ocr_res` <img alt="image" width="957" src="https://private-user-images.githubusercontent.com/2135292/400378371-5f0a784e-1c7c-4228-9d98-017682366f68.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzYxNTkzMjUsIm5iZiI6MTczNjE1OTAyNSwicGF0aCI6Ii8yMTM1MjkyLzQwMDM3ODM3MS01ZjBhNzg0ZS0xYzdjLTQyMjgtOWQ5OC0wMTc2ODIzNjZmNjgucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI1MDEwNiUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNTAxMDZUMTAyMzQ1WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9MzgwOTA0MjE3MGIzZjlhOWMyMTNiY2ZkMjEzZTZmZDhjNzJjMDJhODlkODFjM2ZhMDc2NTllYmJmMGU1MmY0OSZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QifQ.PDd58SyevLmZRsXJalMndkPjq93-2HmZCju6Qyfe1iU">

I have the same problem

**Comment by michaelice** [comment]
@KevinHuSh I tried the updated code in #4387 and encountered a new error, TypeError: ‘super’ object is not callable. Therefore, I submitted a PR in #4404. Now it runs normally and produces the correct output.
![image](https://github.com/user-attachments/assets/1a6b5429-94de-40c2-90b9-16f9100df7c4)
提交了#4404 PR代码后，可以正常执行，结果验证正确。
![image](https://github.com/user-attachments/assets/ffbfd216-1719-4b2e-9e4e-22ccdf1706f0)
![image](https://github.com/user-attachments/assets/a1e042e4-5e33-4fb6-a25f-028952ae1ec8)
